examples/apps/collaborative-textarea/package.json

```diff
@@ -11,13 +11,13 @@
   },
   "license": "MIT",
   "author": "Microsoft and contributors",
-  "main": "dist/index.js",
+  "main": "lib/index.js",
   "module": "lib/index.js",
-  "types": "dist/index.d.ts",
+  "types": "lib/index.d.ts",
   "scripts": {
     "build": "concurrently npm:build:compile npm:lint",
-    "build:compile": "concurrently npm:tsc npm:build:esnext",
-    "build:esnext": "tsc --project ./tsconfig.esnext.json",
+    "build:compile": "npm run build:esnext",
+    "build:esnext": "tsc",
     "build:full": "concurrently npm:build npm:webpack",
     "build:full:compile": "concurrently npm:build:compile npm:webpack",
     "clean": "rimraf dist lib *.tsbuildinfo *.build.log",
```

Comment:
Since we're no longer generating a `dist` directory in these packages, we should probably remove it from `clean`.

Reply:
if we did, people would be left with their old dist directories from before this change. Also I think some of these examples might still put their webpacked results in there, so it seems better to keep it

Reply:
They would, but I would think that would be harmless. Nothing (presumably) would use that output, and a git clean would fix everything. That said, I don't think it's harmful to leave either way.